.travis.yml

```diff
@@ -1,4 +1,7 @@
 language: rust
+rust:
+  - nightly
+  - 1.0.0-beta
```

Comment:
Needs to be `1.0.0-beta.4`. You are referring to the first beta version, is different than the current one.